### PR TITLE
[AlphaMolWrapper] add recipe

### PR DESCRIPTION
**Pull request by IvanSpirandelli**
- Name: AlphaMolWrapper
- Version: v0.1 (446c9949291c89596a58cd14f9864101ff0dc9bf)
- Source: https://github.com/IvanSpirandelli/AlphaMolWrapper
- Language: C++
- BuildSystem: CMake
- Depends: GMP
- Depends (optional): None
- Purpose: Calculation of geometric measures of a union of 3d balls.
- Description: "AlphaMol can calculate volume, surface area, mean curvature and Gaussian curvature of a union of 3d balls. Furthermore it can also compute the derivatives of the measures."

### CODE REVIEW

**Review by giordano — COMMENTED** [review]
Thanks for your contribution!

As a general remark working with Git and GitHub, opening a PR from a branch with the same name as the target branch is an [anti-pattern](https://blog.jasonmeridth.com/posts/do-not-issue-pull-requests-from-your-master-branch/). You should always create a ***new*** branch.

File: A/AlphaMolWrapper/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+platforms = expand_cxxstring_abis(vcat(libjulia_platforms.(julia_versions)...))
+
+products = [
+    LibraryProduct("libalphamolwrapper", :libalphamolwrapper),
+]
+
+dependencies = [
+    BuildDependency("libjulia_jll"),
+    Dependency("libcxxwrap_julia_jll"),
+    Dependency("GMP_jll"),
```
[the remark this answers is not in the document]
Comment (IvanSpirandelli) [reply A/AlphaMolWrapper/build_tarballs.jl]: Setting the it to "6.2.0" gives the following error:

`
ERROR: LoadError: Unsatisfiable requirements detected for package GMP_jll [781609d7]:
GMP_jll [781609d7] log:
├─possible versions are: 6.2.1 or uninstalled
└─restricted to versions 6.2.0 by an explicit requirement — no versions left
`

[the remark this answers is not in the document]
Comment (IvanSpirandelli) [reply A/AlphaMolWrapper/build_tarballs.jl]: So I have to set it to 6.2.1 I assume?

**Comment by IvanSpirandelli** [comment]
Tried setting GMP_jll to both 6.2.0 and 6.1.2. Both threw the following error:

```
ERROR: LoadError: Unsatisfiable requirements detected for package GMP_jll [781609d7]:
GMP_jll [781609d7] log:
├─possible versions are: 6.2.1 or uninstalled
└─restricted to versions 6.2.0 by an explicit requirement — no versions left
```

**Comment by IvanSpirandelli** [comment]
As I cannot seem to use GMP_jll 6.2.0 or less, I can also not have Julia 1.6.3 compat.

**Pull request merged**